core/src/main/scala/com/lightbend/paradox/template/PageTemplateSystem.scala

```diff
@@ -23,7 +23,7 @@ import org.stringtemplate.v4.{ STErrorListener, STRawGroupDir, ST }
 import collection.concurrent.TrieMap
 
 object CachedTemplates {
-  def apply(dir: File, templateName: String): PageTemplate = {
+  def apply(dir: File, templateName: String = PageTemplate.DefaultName): PageTemplate = {
```

Comment:
Look at the code:

```scala
cache.get(templateName) match {
        case Some(t) => t
```

It doesn't care what `directory` you asked for, it caches things using `templateName` as the key.

You're comment only increases my fear and doubt :-/